### ISSUE TITLE
Feature Request: Option to Select Timezone.

### ISSUE THREAD

**Issue by renzms** [wpsharks/comet-cache#895]
It would be nice if a user had the option to use the local server's timezone or had an option to select a timezone. For example, using GMT timezones since the current timezone in the plugin is only UTC.

_____

Referencing Internal ticket: https://secure.helpscout.net/conversation/376029653/1351/

**Comment by CyberCr33p** [comment]
Thank you.

**Comment by raamdev** [comment]
What part of Comet Cache is this referring to? Comet Cache is a WordPress plugin and uses the WordPress date/time settings in **Dashboard → Settings → General**.

**Comment by CyberCr33p** [comment]
The time in the footer of the page:

Cache File Version Salt:       n/a 

Cache File URL:                http://www.xxx.gr/ 
Cache File Path:               /cache/comet-cache-xxx/cache/http/www-xxx-gr/index.html 

Cache File Generated Via:      HTTP request 
Cache File Generated On:       Jun 9th, 2017 @ 9:22 am UTC 
Cache File Generated In:       1.42426 seconds 

Cache File Expires On:         Jun 16th, 2017 @ 9:22 am UTC 
Cache File Auto-Rebuild On:    Jun 16th, 2017 @ 9:22 am UTC 

Loaded via Cache On:    Jun 9th, 2017 @ 9:24 am UTC 
Loaded via Cache In:    0.01759 seconds

**Comment by raamdev** [comment]
@CyberCr33p Ah, I see. Thank you for explaining. Yes, I agree those timestamps should follow whatever your WordPress timezone settings are set to. I'll mark this as an accepted feature request.

**Comment by raamdev** [comment]
@jaswrks It sounds like this might be tricky given the fact that the cache file is served _before_ the rest of WordPress is loaded, which means we can use the usual method of retrieving the timezone info set in WordPress. Thoughts?

**Comment by jaswrks** [comment]
That is correct, yes. Too early to query the database. So to do this it would require saving the configured timezone and hard-coding it into the `advanced-cache.php` file in the form of a constant perhaps.
